### PR TITLE
FAT2-172 add acceptance tests

### PR DESCRIPTION
**Pull request by dashton82**

### CODE REVIEW

File: src/SFA.DAS.FAT.Web.AcceptanceTests/Steps/CoursesSteps.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            {
+                actualContent.Should().Contain(HttpUtility.HtmlEncode(sector.Route));
+            }
+            foreach (var level in expectedApiResponse.Levels)
+            {
+                actualContent.Should().Contain($"Level {level.Code} - {HttpUtility.HtmlEncode(level.Name)}");
+            }
+        }
+
+        [Then("the Api is called with the keyword baker")]
```
Comment (PeterDavidWilliams) [inline src/SFA.DAS.FAT.Web.AcceptanceTests/Steps/CoursesSteps.cs:61]: For future proofing, should this be `the Api is called with the keyword: (*)` and have the course passed as a param?

File: src/SFA.DAS.FAT.Web.AcceptanceTests/Features/CourseDetail.feature [hunk 2]
```diff
@@ -2,28 +2,30 @@
...
 
+@WireMockServer
 Scenario: Regulated course
-When I navigate to the following url: /courses/333
-Then an http status code of 200 is returned
-And the regulated occupation header and message is displayed
\ No newline at end of file
+	Given I navigate to the following url: /courses/333
+	Then an http status code of 200 is returned
+	And the regulated occupation header and message is displayed
```
Comment (PeterDavidWilliams) [inline src/SFA.DAS.FAT.Web.AcceptanceTests/Features/CourseDetail.feature]: If we're checking that the inappropriate content isn't shown in the first test, should we not check that in the other 2 tests here?

File: src/SFA.DAS.FAT.Web.AcceptanceTests/Steps/CoursesSteps.cs (hunk 1 above)
Comment (dashton82) [reply src/SFA.DAS.FAT.Web.AcceptanceTests/Steps/CoursesSteps.cs:61]: ahh good spot, I've updated

File: src/SFA.DAS.FAT.Web.AcceptanceTests/Features/CourseDetail.feature (hunk 2 above)
Comment (dashton82) [reply src/SFA.DAS.FAT.Web.AcceptanceTests/Features/CourseDetail.feature]: I've refactored this so its only checking the one thing that its actually interested in 😄

**Review by PeterDavidWilliams — APPROVED** [review]

**Pull request merged**